rasa/cli/train.py

```diff
@@ -179,10 +179,15 @@ def train_nlu(
 
 
 def extract_additional_arguments(args: argparse.Namespace) -> typing.Dict:
-    return {
+    arguments = {
         "augmentation_factor": args.augmentation,
         "dump_stories": args.dump_stories,
         "debug_plots": args.debug_plots,
-        "percentages": args.percentages,
-        "runs": args.runs,
     }
+
+    if "percentages" in args:
```

Comment:
I'm not sure what you mean. Can you please rephrase?

Reply:
"percentages" is used for the compare training but not for the regular training, right? But I think your code is only used for the regular traininig.

Reply:
The args get passed on to the compare training (https://github.com/RasaHQ/rasa/blob/cli-fix-test-arguments/rasa/cli/train.py#L164). In that method the parameters `percentages` and `runs` are extracted and used. So should be fine.

Reply:
ok, but then you don't need 

```
if "percentages" in args:
        arguments["percentages"] = args.percentages
    if "runs" in args:
        arguments["runs"] = args.runs
``` 
Since they are not used during the regular training, right?